cluster/gce/util.sh

```diff
@@ -1680,7 +1684,9 @@ function generate-certs {
     cd "${CERT_DIR}"
     ./easyrsa init-pki
     # this puts the cert into pki/ca.crt and the key into pki/private/ca.key
+    # shellcheck disable=SC2153
```

Comment:
I know you allude to shellcheck directives in the comment above, but it would be more helpful for future maintainers to call out the specific variable(s) the disable directive is being used each time you use it

Reply:
Agree, I will add such clarifying comments.